R/resource.R

```diff
@@ -0,0 +1,160 @@
+##' Specify what resources a task requires to run.
+##'
+##' # Windows cluster
+##'
+##' The windows cluster currently is all 32-core nodes, with RAM
+##' mainly 512Gb, and a few 384Gb.
+##'
+##'
+##' # Linux cluster
+##'
+##' Coming Soon.
+##'
+##' @title Task Resources
+##'
+##' @param cores The number of cores your task requires. This is 1 by
+##'   default. An error is reported if you request more cores than
+##'   is available on the largest cluster node. Setting to `Inf` will
+##'   request any entire node, however many cores it has.
+##'
+##' @param exclusive Set this to `TRUE` to ensure no other tasks will be
+##'   concurrently run on the node while it runs your task.
+##'
+##' @param runtime Set this to specify a time limit for running your job.
+##'   Acceptable formats are either an integer number of minutes, or
+##'   strings in the form `"h:m"`, or `"d:h:m"` to specify
+##'   days, hours and minutes.
+##'
+##' @param hold_until Specify your task should wait in the queue until
+##'   a certain time, or for a certain period. For the former, this can
+##'   be a POSIXt, a Date, the special strings "tonight" (7pm), "midnight",
+##'   or "weekend" (midnight Saturday morning). To delay for a period,
+##'   this can be an integer number of minutes, or a string in the form
+##'   `"h:m"` for hours and minutes.
```

Comment:
```suggestion
##'   `"h:m"` for hours and minutes, as for `runtime`.
```

(or `runtime_limit` if you choose to rename that)